### PR TITLE
Update Standard to 0.10.2 and fix offenses

### PR DESCRIPTION
**Pull request by natematykiewicz**

### CODE REVIEW

File: app/services/search/filters/is.rb [hunk 1]
```diff
@@ -13,9 +13,9 @@ def self.filter_for(value)
           [{term: {object_type: {value: :class_object}}}]
         when "method"
           [{term: {type: {value: :method}}}]
-        when "class-method", "class-method", "cmethod"
+        when "class-method", "cmethod"
           [{term: {method_type: {value: :class_method}}}]
-        when "instance-method", "instance-method", "imethod", "#"
```
Comment (colby-swandale) [inline app/services/search/filters/is.rb:18]: These values are not suppose to change, they're keywords used to filter search results.

Comment (natematykiewicz) [reply app/services/search/filters/is.rb:18]: instance-method and class-method are listed twice. It's my understanding that this diff executes exactly the same as the old code. The duplication is why the Deendabot PR failed checks.

Comment (colby-swandale) [reply app/services/search/filters/is.rb:18]: Oh I see, not sure why I did this 🤔

**Review by colby-swandale — APPROVED** [review]

File: app/services/search/filters/is.rb (hunk 1 above)
Comment (natematykiewicz) [reply app/services/search/filters/is.rb:18]: Not sure. I tried following it back in the commits but it looks like you renamed the file at one point. It was there in the original commit of this file.

All I can guess is that you renamed some method in a refactor, and did a find/replace on the method name, which also updated one of the values in the case statement, creating a duplicate.

**Pull request merged**